scripts/stress-all.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+thisdir=`dirname "$0"`
+set -euo pipefail
+
+${thisdir}/infra.sh -s
+
+killApp () {
+  (lsof -t -i:8080 || true) | xargs -r kill
+}
+
+wait_for_8080() {
+    echo "Waiting for port 8080..."
+    for ((i=0; i<30; i++)); do
+        # Using 127.0.0.1 is safer than localhost on macOS to avoid IPv6 ::1 mismatch
+        if (echo > /dev/tcp/127.0.0.1/8080) >/dev/null 2>&1; then
+            return 0
+        fi
+        sleep 1
+    done
+    echo "Timeout waiting for port 8080"
+    return 1
+}
```

Comment:
@quintesse have a look also at this script, could be useful for your own; this one waits just enough for the frameworks to have opened port 8080.  Seems like a good replacement for the "10 seconds" rule.

Also, you could then use the total time of the benchmark - including boostrap times - as a fair comparison.